internal/report/slo.go

```diff
@@ -1149,6 +1149,52 @@ and check the logs for errors.`,
 and check the logs for errors.`,
 		},
 	})
+	checkSum.Checks = append(checkSum.Checks, &Check{
+		ID:   "OPCT-040",
+		Name: "RHOCP Validation: cluster must be installed with supported platform type",
+		Test: func() CheckResult {
+			res := CheckResult{
+				Name:   CheckResultNameFail,
+				Target: "install/PlatformType",
+			}
+			if re.Provider == nil || re.Provider.Infra == nil {
+				return res
+			}
+			// Evaluate valid providers
+			res.Actual = fmt.Sprintf("%s/%s", *re.Provider.InstallInvoker, re.Provider.Infra.PlatformType)
+
+			// TODO: "assisted-installer/None" must be removed when
+			// Assisted Service UI allows to install platform type External to any provider.
+			// Currently only Oracle Cloud allows to install "assisted-installer/External".
+			switch res.Actual {
+			case "user/External", "assisted-installer/External", "assisted-installer/None", "agent-installer/External":
```

Comment:
Do we need to set to Warn when using `"assisted-installer/None"`. IMHO it would be complicated as currently Assisted UI does not support "assisted-installer/External"